File: pset1/cash/cash.c
#include <stdio.h>
#include <cs50.h>
#include <math.h>

int main(void)
{
    float change = -1;
    while (change < 0)
    {
        change = get_float("Change owed: ");
    }
    int cents = round(change * 100);
    int quarters = cents / 25;
    cents = cents - quarters * 25;
    int dimes = cents / 10;
    cents = cents - dimes * 10;
    int nickels = cents / 5;
    cents = cents - nickels * 5;
    int pennies = cents;
    int total = quarters + dimes + nickels + pennies;
    printf("%d\n", total);
}

File: pset6/cash/cash.py
import cs50

change = -1
while True:
    try:
        change = cs50.get_float("Change owed: ")
        if 0 <= change:
            break
    except ValueError:
        continue

cents = int(change * 100)
quarters = int(cents / 25)
cents = int(cents - quarters * 25)
dimes = int(cents / 10)
cents = int(cents - dimes * 10)
nickels = int(cents / 5)
cents = int(cents - nickels * 5)
pennies = int(cents)
total = quarters + dimes + nickels + pennies
print(int(total))